src/main/java/org/spongepowered/mod/SpongeMod.java

```diff
@@ -134,10 +107,25 @@ public void onPreInit(FMLPreInitializationEvent e) {
     @Subscribe
     public void onInitialization(FMLInitializationEvent e) {
         this.registry.init();
+        try {
+            getGame().getServiceManager().setProvider(this, CommandService.class, new SimpleCommandService(getGame().getPluginManager()));
```

Comment:
Where? That's part of the API: `https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/service/SimpleServiceManager.java#L67`